### ISSUE TITLE
Render model for constructionsite

### ISSUE THREAD

**Issue by thmsndk** [thmsndk/Screeps3D#97]
As an added visual to just rendering a circle with progress we had a chat on slack and came up with the following

- Render the model in a greenish color / shade
- Mask the model bottom -> top depending on progress

We could perhaps indicate if a construction site is walkable or not.

> walkable ones are roads, ramparts, containers. Roads we could keep as they are right now, I don't think there is a better way to visualize their progress. Containers are pretty flat anyway, ramparts take one build to be created, we can keep them as they are or make some box wireframe

construction sites with owner info can also be colored red instead of green if not owned by you, perhaps tint the ones without owner info yellow?

**Comment by M1kep** [comment]
What if the construction site was the model that was gonna be built. But had some sort of shader and would be built in pieces?

**Comment by Gadjung** [comment]
i've did quick check, and it would be doable to load proper prefab - would require though modification of material to 'hologram' like for example - which i did not manage to do

**Comment by thmsndk** [comment]
@Gadjung Yeah, considered keeping the circle as well for resemblance, but having the csite beign built up based on progress would be cool :)
